community/concepts/cred.md

```diff
@@ -0,0 +1,20 @@
+# Cred
+
+SourceCred, as the name might suggest, is all about attributing cred. Cred is a metric that describes every contribution and contributor in a project to give a sense of how important they were.
+
+For example, forum posts can earn cred. If I post on the [sourcecred discourse](https://discourse.sourcecred.io/) and earn 5 cred, and another user posts and their post earns 10 cred, we can say that the other post is considered twice as important as my post.
+
+The contributions are arranged in a [graph](https://en.wikipedia.org/wiki/Graph_(discrete_mathematics)) where contributions are nodes, and have edges indicating how they relate to other contributions. For example, if someone writes a reply to my post, there will be an edge connecting the reply and my post. Edges have "types" that give some information about what kind of edge they are; that edge might have an "IS_REPLY" type. The connections are also directional, meaning their are relationships that connect nodes to it, and relationships that connect nodes from it. An edge without a direction is considered bi-directional.
+
+Contributors — like you or me — are also nodes in the graph, and are connected to the contributions that they create. So there is an "AUTHORS" edge connecting me (the author) to my post.
+
+SourceCred converts this graph into a numerical score via [PageRank](https://en.wikipedia.org/wiki/PageRank). Basically, we assign cred to the nodes so that every node receives cred from every node that connects to it, and in turn sends its cred to every node that connects from it. This means that cred accumulates at important nodes. For example, a core maintainer is connected to all of the posts, comments, and issues that they’ve written, so they have a lot of cred. On the other hand, a spam post on the forum may be sparsely connected, or connected to almost no other nodes, so it will have very little cred. (PageRank is a very interesting algorithm, and was actually the basis of Google search! If you want to learn more, I recommend the original [PageRank paper](http://ilpubs.stanford.edu:8090/422/1/1999-66.pdf)).
```

Comment:
>and in turn sends its cred 

'it's cred' could be read by some as 'all of its cred'. Do we mean 'sends some of its cred'? Or am I misunderstanding the algorithm?

Reply:
I think it's saying that "Here we have a node, it has X cred" and the implication is that the X cred is divided between "every node that connects to it" but I agree it would be more clear to explicitly state this.

Reply:
I'm going to leave this unresolved because I'm not sure what the right answer is.